### PR TITLE
New documentation layout (for desktops)

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 

That's what you've been seeing on the website for several weeks already - a scrollable menu on the left, the topic occupies most of the remaining space.

[http://strelyaev-w8:8080/testcafe/documentation/getting-started/](http://strelyaev-w8:8080/testcafe/documentation/getting-started/)

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
lgtm

File: src/_includes/head.html [hunk 1]
```diff
@@ -34,24 +37,105 @@
...
+    <script defer type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery.perfect-scrollbar/0.6.12/js/perfect-scrollbar.min.js"></script>
+    <script>
+        function initNav(nav) {
+            var MIN_BODY_WIDTH = 900;
+            var FOOTER_HEIGHT  = 63;
+            var HEADER_HEIGHT  = 52;
 
-          if (!siteHeader)
-              return;
+            function updatePosition() {
```
Comment (kirovboris) [inline src/_includes/head.html]: I think this function can be significant refactored. At first sight:
```javascript
function updatePosition() {
	var footerVisibleHeight = FOOTER_HEIGHT - document.body.scrollHeight + window.pageYOffset + document.body.clientHeight;
	nav.style.marginLeft    = '-' + window.pageXOffset + 'px';

	if (footerVisibleHeight > 0 && nav.style.bottom !==  footerVisibleHeight + 'px') 
		nav.style.bottom =  footerVisibleHeight + 'px';
	else if(nav.style.bottom !== '0') 
		nav.style.bottom = '0';	
	
	if (document.body.clientWidth > MIN_BODY_WIDTH && nav.style.top !== HEADER_HEIGHT + 'px') 
		nav.style.top = HEADER_HEIGHT + 'px';
	else {
		var top = window.pageYOffset <= HEADER_HEIGHT ? (HEADER_HEIGHT - window.pageYOffset) + 'px' : '0';
		
		nav.style.top = top;
	}
	
	Ps.update(nav);		
}
```

Comment (kirovboris) [inline src/_includes/head.html]: Moreover you don't need script for this behavior. Just put
```javascript
<div class="push"></div>
<footer class="site-footer">
<div class="site-footer-content">
<div class="footer-col-1">
	<p><strong>TestCafe</strong>: Automated browser testing for the modern web development stack<br>
	Copyright © 1998-2016 Developer Express Inc.</p>
</div>
</div>

```
a the end of ```<article>``` and add ```position: absolute, left: 0, right: 0``` to ```.site-footer```. Also you can add  ```min-width ``` for  ```.site-footer```

Comment (VasilyStrelyaev) [inline src/_includes/head.html]: There are three things scripted here:

1) let the navigation scroll to the left if the content is not completely visible.
2) let the header push the navigation down when the former is not fixed
3) let the footer push the navigation up when the former is visible

Are you talking only about number 3?

Comment (VasilyStrelyaev) [inline src/_includes/head.html]: Not to say that I'm not sure that the footer is semantically a part of `<article>`.

Comment (kirovboris) [inline src/_includes/head.html]: About 3 only. You can create container for ```<article>``` and ```<footer>```

**Comment by kirovboris** [comment]
lgtm, We discussed offline and decide not refactor it.

**Pull request merged**